### DISCUSSION TITLE
thanhle.blog/nang-cap-tu-dev-frontend-web2-sang-web3

### DISCUSSION THREAD

**Discussion by giscus** [thanhlmm/blog#58] [Ideas]
# thanhle.blog/nang-cap-tu-dev-frontend-web2-sang-web3

Cách năng cấp bản thân để phù hợp với thời đại mà không tốn quá nhiều công sức

https://thanhle.blog/blog/nang-cap-tu-dev-frontend-web2-sang-web3

**Comment by cuongnc3010** [comment]
Bài viết quá phê, mình có thể liên hệ ad thông qua email hoặc kênh chat, mạng xã hội nào ko?

**Reply by thanhlmm** [reply]
Thanks bạn, contact mình ở https://thanhle.blog/me hoặc FB https://www.facebook.com/thanh.lmm/ nhé

**Reply by cuongnc3010** [reply]
Mình đã pm facebook, ad add mình được ko?! Cám ơn ad!

**Comment by thanhlmm** [comment]
A new place to learn Web3 https://university.alchemy.com/ 🚀

**Comment by mioriaty** [comment]
Post blogchain cho mọi người hình như bị xoá rồi à anh

**Reply by thanhlmm** [reply]
hình như do API notion bị limit nến không hiện ra ấy. E check thử ở đây nhé https://thanh-le.notion.site/c0a9456d6fa04bb2af554a310ac7b5ff?v=34aad7277ebc4924ae16a897baad7849&pvs=74

Mà bài nào e nhỉ?

**Reply by mioriaty** [reply]
em cảm ơn ạ,bị 404 https://thanhle.blog/blog/blockchain-cho-moi-nguoi bài này anh ạ
